packages/dds/tree/src/feature-libraries/sequence-field/invert.ts

```diff
@@ -87,6 +84,29 @@ function invertMark<TNodeChange>(
                     },
                 ];
             }
+            case "MoveOut":
+            case "ReturnFrom": {
+                return [
```

Comment:
We also need to inver the `mark.changes` if any.
The idea of having detach marks in detached trees and attach marks in attached trees was in part motivated by the fact that it simplifies invert.

Reply:
Ah, right. My preference is to just to transfer the child marks. It should be very simple compared to the move handling in rebase and compose, and I'd rather have a little extra complexity in `invert` than extra complexity in the contract for the format.

Reply:
Sounds good.